lib/hammer_cli_foreman/compute_resource/libvirt.rb

```diff
@@ -48,7 +49,8 @@ def provider_vm_specific_fields
             Fields::Field.new(:label => _('Status'), :path => [:state]),
             Fields::Field.new(:label => _('OS Type'), :path => [:os_type]),
             Fields::Field.new(:label => _('Domain Type'), :path => [:domain_type]),
-            Fields::Field.new(:label => _('Persistent'), :path => [:persistent])
+            Fields::Field.new(:label => _('Persistent'), :path => [:persistent]),
+            Fields::Field.new(:label => _('Boot order'), :path => [:boot_order])
```

Comment:
Oh, sorry, I forgot that this will actually be a list (array), so it's better to use `Fields::List` here.